### DISCUSSION TITLE
Tickers from Xetra not available?

### DISCUSSION THREAD

**Discussion by mchodakowski** [ranaroussi/yfinance#1582] [Q&A]
Hi,
I've just noticed that I can't get quotes from Xetra. For example when I'm requesting for W1TA.DE I'm getting the following error:
_- W1TA.DE: No data found for this date range, symbol may be delisted_ I've looked on Yahoo Finance and I think, the ticket is not delisted: https://finance.yahoo.com/quote/W1TA.DE?p=W1TA.DE&.tsrc=fin-srch I've check few more *.DE tickers and none of them seems to work. Just two days ago everything was OK. Have you noticed such problem?

**Comment by samo-L-B** [comment]
same probleme here - even though some stocks like e.g. EVT.DE can be accessed but others (mostly small cap stocks) could not imported.

Have you found an approach to import european/german stocks in a reliable way?
